src/types/globals/l10n.js

```diff
@@ -0,0 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+// This is a variable set by Webpack when we are in the l10n branch.
```

Comment:
nit: similar comment: `when we run it with L10N=1, especially in the l10n branch`